README.md

```diff
@@ -27,6 +27,11 @@ The Nimiq Rust client comes without wallet and can currently not be used to send
 - [Nimiq Developer Reference](https://nimiq-network.github.io/developer-reference/): Details of the protocol architecture.
 
 
+## System requirements
+- 32-bit architectures are not supported.
+- File systems with sparse file support (most modern files systems support it).
```

Comment:
Do these requirements only apply when the size of 1TB is used? If a config file specifies a smaller file size, can 32-bit systems without sparse file support still work?

Reply:
Yes, but the limit would be 4G which we think we can quickly reach according to the initial data we have collected.